website/static/js/citationGrid.js

```diff
@@ -355,7 +355,10 @@ CitationGrid.prototype.makeBibliography = function(folder) {
             return child.data.csl;
         })
     );
-    var citeproc = citations.makeCiteproc(this.styleXml, data, 'html');
+    if (!format) {
```

Comment:
I prefer `format = format || 'html';` - but it's correct either way and I believe our style guide is silent on the matter.